src/components/laser-controls.js

```diff
@@ -0,0 +1,44 @@
+var registerComponent = require('../core/component').registerComponent;
+var utils = require('../utils/');
+
+registerComponent('laser-controls', {
+  schema: {
+    hand: {default: 'right'}
+  },
+
+  init: function () {
+    var config = this.config;
+    var data = this.data;
+    var el = this.el;
+
+    el.setAttribute('daydream-controls', {hand: data.hand});
+    el.setAttribute('gearvr-controls', {hand: data.hand});
+    el.setAttribute('oculus-touch-controls', {hand: data.hand});
+    el.setAttribute('vive-controls', {hand: data.hand});
+
+    el.addEventListener('controllerconnected', function (evt) {
+      el.setAttribute('cursor', utils.extend({
```

Comment:
You're probably aware but we still need the logic here to position the laser based on the controller